spec/models/web_hook_events_daily_aggregate_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+RSpec.describe WebHookEventsDailyAggregate do
+  let(:web_hook) { Fabricate(:web_hook) }
+  let(:event) { WebHookEvent.new(status: 200, web_hook: web_hook, created_at: 2.days.ago) }
+  let(:failed_event) { WebHookEvent.new(status: 400, web_hook: web_hook, created_at: 2.days.ago) }
+
+  describe "aggregation works" do
```

Comment:
I think this needs to be more comprehensive, testing multiple webhooks on each day and also testing that the successful/failed counts and the mean_duratiion are all calculated correctly